OJP/OJP_ModesSupport.xsd

```diff
@@ -8,19 +8,43 @@
 	<xs:annotation>
 		<xs:documentation>========================================== Individual Modes ==========================================</xs:documentation>
 	</xs:annotation>
-	<xs:simpleType name="IndividualModesEnumeration">
+	<xs:simpleType name="IndividualModesOfOperationEnumeration">
 		<xs:annotation>
 			<xs:documentation>modes which an individual powers themselves (such as walk, cycle)</xs:documentation>
 		</xs:annotation>
 		<xs:restriction base="xs:string">
 			<xs:enumeration value="walk"/>
 			<xs:enumeration value="cycle"/>
-			<xs:enumeration value="taxi"/>
 			<xs:enumeration value="self-drive-car"/>
 			<xs:enumeration value="others-drive-car"/>
+			<xs:enumeration value="vehicleHire">
+				<xs:annotation>
+					<xs:documentation>One of the MODES OF OPERATIOn from NeTEX rental.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="vehicleLease">
+				<xs:annotation>
+					<xs:documentation>One of the MODES OF OPERATIOn from NeTEX rental.</xs:documentation>
+				</xs:annotation>			
+			</xs:enumeration>			
 			<xs:enumeration value="motorcycle"/>
 			<xs:enumeration value="truck"/>
 			<xs:enumeration value="scooter"/>
+			<xs:enumeration value="carSharingClub">
+				<xs:annotation>
+					<xs:documentation>Car sharing club. MODES OF OPERATIOn from NeTEx sharing.</xs:documentation>
```

Comment:
```suggestion
					<xs:documentation>Car sharing club. MODES OF OPERATION from NeTEx sharing.</xs:documentation>
```

Reply:
I even copy&paste my errors.

Reply:
At least your MODE OF OPERATION is consistent ;)